examples/gepetto-viewer.py

```diff
@@ -11,8 +11,8 @@
 # Load the URDF model.
 model_path = Path(os.environ.get("EXAMPLE_ROBOT_DATA_MODEL_DIR"))
 mesh_dir = model_path.parent.parent
-urdf_filename = "talos_reduced.urdf"
-urdf_model_path = model_path / "talos_data/robots" / urdf_filename
+urdf_filename = "panda.urdf"
+urdf_model_path = model_path / "panda_description/urdf" / urdf_filename
```

Comment:
I guess you only changed that to debug, but there is no need for this change in this PR